### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/locales/fa/app.json [hunk 1]
```diff
@@ -873,7 +873,7 @@
   },
   "loading": "در حال بارگذاری…",
   "notification": {
-    "newTxs_0": "{{count}} معاملات جدید در: {{accountName}}"
+    "newTxs_one": "{{count}} معاملات جدید در: {{accountName}"
```
Comment (thisconnect) [inline frontends/web/src/locales/fa/app.json:876]: `}` missing, I fixed this in locize

**Review by thisconnect — CHANGES_REQUESTED** [review]
Please pull again.

File: frontends/web/src/locales/fa/app.json (hunk 1 above)
Comment (benma) [reply frontends/web/src/locales/fa/app.json]: good catch, pulled again

**Review by thisconnect — APPROVED** [review]
LGTM

**Pull request merged**